packages/blade/src/components/Radio/Radio.tsx

```diff
@@ -51,6 +63,8 @@ const Radio = ({ value, children, helpText, isDisabled }: RadioProps): React.Rea
   const name = groupProps?.name;
   const showHelpText = !hasError && helpText;
   const isReactNative = getPlatformType() === 'react-native';
+  const _size = groupProps.size ?? size;
+  const isSmall = _size === 'small';
```

Comment:
We're going to have `large` as well now. This logic will need to change and a boolean won't be enough :(

Upto you if you want to prepare for large size in this PR or the following PR for large size

Reply:
I'll do it on the next PR. and refactor it accordingly.